api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -34,7 +152,7 @@
  *      1.1 Primer</a>
  * @see org.apache.commons.rdf.experimental.RDFParser
  */
-public enum RDFSyntax {
+  public enum OfficialRDFSyntax implements RDFSyntax {
```

Comment:
Added RDFSyntax.iri() in 81d5b32 with a simplified private `IRI` implementation. I also changed the `.name()` of w3c syntaxes to match the suffix of the format URI.

(This means there are currently two entries for `RDFSyntax.name()=="RDFa"`)